tests/peerconnection.js

```diff
@@ -1976,6 +1976,23 @@ describe('PeerConnection', () => {
       }).then(done).catch(done);
     });
 
+    it('Should call setSinkId with \'\' if empty string is first device', done => {
+      output.audio.setSinkId.returns(Promise.resolve());
```

Comment:
If I'm not mistaken, this will cause the function to return the same promise every time. I don't see it affecting anything in this case, but wanted to note it.